src/taoverse/model/model_updater.py

```diff
@@ -107,8 +113,8 @@ async def sync_model(
         # If not we return false and will check again next time we go through the update loop.
         if curr_block - metadata.block < competition.constraints.eval_block_delay:
             bt.logging.debug(
-                f"""Sync for hotkey {hotkey} delayed as the current block: {curr_block} is not at least 
```

Comment:
Removing this space will make this log harder to read since otherwise the next line happens with no break.

Reply:
Good catch